routes/rendering-info/html-static.js

```diff
@@ -98,9 +99,9 @@ module.exports = {
     }
 
     if (isSophieVizColorDefined) {
-      responseData.stylesheets.push({
-        url:
-          "https://service.sophie.nzz.ch/bundle/sophie-viz-color@^1.0.0[parties].css"
+      responseData.sophieModules.push({
+        name: "sophie-viz-color@^1.0.0",
```

Comment:
You can declare the version like `sophie-viz-color@1`. Like this is will always get the newest version starting with 1.